build.rs

```diff
@@ -19,7 +19,19 @@ fn download_filename() -> String {
 }
 
 fn get_expected_sha256(filename: &str) -> Result<sha256::Hash, ()> {
-    let sha256sums_filename = format!("sha256/bitcoin-core-{}-SHA256SUMS.asc", &VERSION);
```

Comment:
can we use the `.asc` version also for `22.0`?

Reply:
I couldn't decide about this, starting with `22.0` two files are created: `SHA256SUMS` and `SHA256SUMS.asc`. The first file has all the checksums and the second file only has signatures. Two other options I see are:
1. for all new releases of `bitcoind` we rename the `SHA256SUMS` file to `SHA256SUMS.asc`, but usually the `.asc` extension means that it's ascii armored which the new release files technically won't be. 
2. we rename all the files in the `sha256` directory, removing the `.asc` extension and we should also remove the signatures so the file names won't be misleading and since we don't use the signatures anyway.

Disadvantage of options 1 is some misleading file naming that conflict with actual files released with that name.  I'd prefer option 2 since going forward for new releases we can just rename and drop in the unedited `SHA256SUMS` file.

Reply:
Sorry, I didn't know the `SHA256SUMS.asc` format changed... Let's keep it as it is in current PR